core/src/templates/group.ts

```diff
@@ -32,8 +35,8 @@ export class GroupTemplate extends ConfigTemplate {
     this.destinationDir = "groups";
   }
 
-  async run({ params }) {
-    if (params.rules) params.rules = JSON.parse(params.rules);
+  async run({ params }: { params: ConfigTemplateParams }) {
+    if (params.rules) params.rules = JSON.parse(params.rules.toString());
```

Comment:
Why not format rules to string?

Reply:
These templates are all going away soon...